Examples/VaporExample/Package.swift

```diff
@@ -1,11 +1,14 @@
-// swift-tools-version:4.2
+// swift-tools-version:5.0
 import PackageDescription
 
 let package = Package(
     name: "VaporExample",
+    platforms: [
+        .macOS(.v10_14)
+    ],
     dependencies: [
-        .package(url: "https://github.com/vapor/vapor", .upToNextMajor(from: "3.3.0")),
-        .package(url: "https://github.com/mongodb/mongo-swift-driver", .upToNextMajor(from: "0.1.0"))
+        .package(url: "https://github.com/vapor/vapor", .exact("4.0.0-beta.3.24")),
```

Comment:
We may want to indicate in the somewhere (e.g. the source of this example, this example's readme, or the project's readme itself) that we only support Vaptor 4 due to Swift NIO dependencies. Seeing as Vapor might win out as the de facto swift web server, we probably want to treat compatibility with it with special care.

Reply:
I've added mention of this to the example's README and to the `Package.swift` file. I'm currently working on a separate PR to update the driver's README and I will mention it there as well.